src/assets/css/themes/darkly-red.css

```diff
@@ -3936,6 +3941,9 @@ input[type="submit"].btn-block {
   .navbar-expand-md .navbar-toggler {
     display: none;
   }
+  .nav-link {
```

Comment:
There are styles for this further up in the file.

Reply:
not within this media query